client/src/components/connect_email.js

```diff
@@ -0,0 +1,34 @@
+const nodemailer = require('nodemailer');
```

Comment:
since this is a server side script. 
please remove it from the client folder and put it on the server side.

Reply:
ok, will do